spec/js/utility/formatters.spec.html

```diff
@@ -0,0 +1,25 @@
+<html>
+    <head>
+      <meta charset="utf-8">
+      <title>Mocha Tests</title>
+      <link href="https://cdn.rawgit.com/mochajs/mocha/2.2.5/mocha.css" rel="stylesheet" />
+    </head>
+    <body>
+      <div id="mocha"></div>
+
+      <script src="https://cdn.rawgit.com/jquery/jquery/2.1.4/dist/jquery.min.js"></script>
+      <script src="https://cdn.rawgit.com/Automattic/expect.js/0.3.1/index.js"></script>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/sinon.js/3.3.0/sinon.min.js"></script>
+      <script src="https://cdn.rawgit.com/mochajs/mocha/2.2.5/mocha.js"></script>
+
+      <script>mocha.setup('bdd')</script>
+      <script src="../../../js/utility/namespacer.js"></script>
```

Comment:
For the sake of readability, would you break these up into sections for dependencies (namespacer), scripts, and specs? _I_ know which is which, but if I get hit by a truck, which tends to happen around here, whoever's next might not be too sure what's up.